pre_commit_hooks/check_munki_pkgsinfo.py

```diff
@@ -210,17 +216,18 @@ def main(argv=None):
                 retval = 1
 
         # Ensure all pkginfo scripts have a proper shebang.
-        shebangs = (
+        builtin_shebangs = [
             "#!/bin/bash",
             "#!/bin/sh",
             "#!/bin/zsh",
             "#!/usr/bin/osascript",
             "#!/usr/bin/perl",
-            "#!/usr/bin/python",
+            "#!/usr/bin/python3",
```

Comment:
I'd like to keep `/usr/bin/python` in there because some old scripts may have been written assuming before macOS 12.3.

Reply:
okay I'll update.